docs/001_develop/03_client-capabilities/008_interaction/checkbox.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Checkbox'
+sidebar_label: 'Checkbox'
+id: client-interaction-checkbox
+keywords: [interaction, interactive, checkbox, box, boolean, form, click, change, checked, toggle, submit, data]
+tags:
+- interaction
+- interactive
+- checkbox
+- box
+- boolean
+- form
+- click
+- change
+- checked
+- toggle
+- submit
+- data
+sidebar_position: 8
+---
+
+import CheckboxDemo from '../../../../examples/ui/client-capabilities/interaction/checkbox.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Checkbox
+
+A form-connected implementation of a [checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) input.
+
+## Example
+
+<CheckboxDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
+Declaration
+```typescript
+<rapid-checkbox></rapid-checkbox>
```

Comment:
Can we have a container around this so we can make it more prominent

Reply:
We could but I would like to discuss your thought process for this? I already feel it's pretty prominent on the page, and it's already pretty significant I am repeating the definition by pulling it out of the fuller example to give it its own code section.

Reply:
These code examples should they be declared as `html` instead of typescript?

Reply:
I was on with Shahin revieiwing it and we were just thinking to make it pop out a bit more. Even if just a border around it like
https://material.angular.io/components/checkbox/overview
[https://primeng.org/checkbox
I see you've added a background hich looks](https://primeng.org/checkbox#basic)

Reply:
Can I just double check we're talking about the same thing here. You've commented on the decleration snippet which is this code block
![Screenshot 2024-10-31 at 22 05 30](https://github.com/user-attachments/assets/7736cd10-f53e-47cb-99ca-1733d2265880)

This is NOT the live example code which is showing the checkbox you can interact with 
![Screenshot 2024-10-31 at 22 05 37](https://github.com/user-attachments/assets/04d975c0-eeb2-4345-937f-286fa79e0d62)

If you were in fact asking about the live example checkbox, as you can see from that picture I have already changed it to stand out on the page more as other people have been asking for it. I presume that is what you mean as the links you've given above all seem to link to the live example rather than just the decleration of the component. Let me know if my assumption is wrong and we can discuss making the code block stand out even more if you'd like.

Reply:
> These code examples should they be declared as html instead of typescript?

I presume you mean just the decleration part as the full example needs to be typecript, but I have changed that and I agree it's better as it's given slightly better syntax highlighting which makes it pop on the page

Reply:
Sorry yes I did just mean the live example checkbox. Yes this stands out more and looks better now, thanks